src/vendor/wordpress/gutenberg.stories.mdx

```diff
@@ -1,7 +1,17 @@
 import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, select } from '@storybook/addon-knobs';
 import codeBlockDemo from '../../design/typography/demo/code-block.twig';
-
-<Meta title="Vendor/WordPress/Gutenberg" />
+import blockImageDemo from './demo/image.twig';
+const alignmentClasses = {
+  None: null,
```

Comment:
Storybook select values are converted to strings no matter what they are, so (annoyingly) `null` is the same as passing in a string called `'null'`. You _can_ pass in an empty string, though, (`''`), which should have the desired effect.

(It's times like these I wish knobs were better documented... but they've been replaced by a more elegant solution in the latest Storybook release, so that's nice!)

Reply:
Thank you, that was extremely helpful. I had originally started with an empty string, but ended up mimicking what I saw in the [documentation I was reading](https://github.com/storybookjs/storybook/tree/master/addons/knobs#select), which used `null`. 🙃